packages/uniforms-antd/__tests__/ListField.tsx

```diff
@@ -160,3 +160,25 @@ test('<ListField> - renders correct error style (with specified style prop)', ()
     '8px',
   );
 });
+
+test('<ListField> - renders proper number of optional values after add new value (with initialCount)', () => {
+  const element = (
+    <ListField name="x" initialCount={3} label="ListFieldLabel" />
+  );
+  const onChange = jest.fn();
+  const wrapper = mount(
+    element,
+    createContext(
+      { x: { type: Array, optional: true }, 'x.$': { type: String } },
+      { onChange },
+    ),
+  );
+
+  expect(wrapper.find(ListAddField).simulate('click')).toBeTruthy();
+  expect(onChange).toHaveBeenNthCalledWith(1, 'x', [
+    undefined,
+    undefined,
+    undefined,
+    undefined,
+  ]); // renders ( initialCount + 1 ) default values
```

Comment:
1. I'd remove this comment.
2. Could you add a test with a different default value as well?